app/views/content/is-teaching-right-for-me/_facts_and_figures.html.erb

```diff
@@ -0,0 +1,30 @@
+<article>
+  <section class="container">
+      <h2 class="heading--box-blue">Facts and figures</h2>
+
+      <div class="inset">
+        <%= render Content::FactListComponent.new(
+          [
+            {
+              icon: "icon-money",
+              value: "£28k",
+              title: "minimum starting salary",
+              link: link_to("Salaries and benefits", page_path("salaries-and-benefits")),
+            },
+            {
+              icon: "icon-money",
+              value: "2 years",
+              title: "support for early career teachers",
+              link: link_to("Early career teachers", page_path("support-for-early-career-teachers")),
+            },
+            {
+              icon: "icon-money",
+              value: "£10k+",
+              title: "Bursaries and scholarships available",
+              link: link_to("Scholarships and bursaries", page_path("funding-and-support/scholarships-and-bursaries")),
```

Comment:
@ethax-ross this third box is slightly bigger than the other two? not sure if it's because the text is slightly longer?

Reply:
have done, and it's still making it longer 🤔